build.gradle.kts

```diff
@@ -30,15 +52,25 @@ dependencies{
     implementation(project(":java"))
     implementation(project(":python"))
     implementation(project(":rider"))
+// todo: recommended by jetbrains but has a bug that openiong classes opens the class file instead of source file
+//    implementation(project(":ide-common", "instrumentedJar"))
+//    implementation(project(":java", "instrumentedJar"))
+//    implementation(project(":python", "instrumentedJar"))
+//    implementation(project(":rider", "instrumentedJar"))
```

Comment:
don't we need those dependencies?

Reply:
Its the same dependencies, they recommend using the instrumented jar but they have a bug, so its in comment until they fix the bug.